templates/.nycrc.tmpl

```diff
@@ -1,5 +1,4 @@
 {
-	"require": ["babel-register"],
-	"sourceMap": false,
-	"instrument": false
+	"exclude": ["test/**", "**/*.d.ts"],
```

Comment:
I think it's too early to change this nycrc now since other packages are using it.
Maybe it's good to create another one and symlink it

Reply:
PR #814